app/pages/profile/feed.cjsx

```diff
@@ -44,6 +44,7 @@ module.exports = React.createClass
           @setState({comments})
 
   render: ->
+    console.log @props.params
```

Comment:
Just an FYI, the react dev tools for chrome are useful for checking a component's current state and props.